package.json

```diff
@@ -17,7 +17,9 @@
     "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
     "test": "jest -c jest.config.ts",
     "test:cov": "jest -c jest.config.ts --coverage",
-    "test:integration": "jest -c jest.integration.config.ts",
+    "test:integration-dev": "jest --setupFiles ./.integration.test-dev.env -c jest.integration.config.ts",
```

Comment:
Would this work?

`"test:integration":  "jest --setupFiles ${SETUP_FILE:-'./.integration.test-dev.env'} -c jest.integration.config.ts"`

That way we don't need a script per environment, and you can just change the setup file with an env var e.g. 
`SETUP_FILE='./.integration.test-staging.env' yarn test:integration`

Reply:
Or even better:
```
"test:integration": "jest --setupFiles ./integration.test-${ENV:-build}.env -c jest.integration.config.ts"
```

Then it would just be:
```
ENV=staging yarn test:integration
```

I suggest using build as the default as this will be the most up to date main stack. The dev environment will only have feature branches, so values will change all the time.

>  Although, I'm not sure how we'd set up environment variables in an environment like staging, where I believe we would not be using Github actions.

Don't worry about this, we won't be using GitHub actions at all to run integration tests. These need to be run within an AWS pipeline, which I will sort out post MVP.

Reply:
There may also be the need to run against other envs from local machine. One would need to remember to change the environment variable.